src/coinbase/types.ts

```diff
@@ -346,6 +346,15 @@ export enum ServerSignerStatus {
   ACTIVE = "active_seed",
 }
 
+/**
+ * Options for creating a Wallet.
+ */
+export type WalletCreateOptions = {
+  networkId?: string;
+  intervalSeconds?: number;
+  timeoutSeconds?: number;
+};
+
```

Comment:
nice - we should make a pass and do this options type arg pattern for all methods that have optional args so they are not order dependent. This doesn't have to be done in this PR!